### DISCUSSION TITLE
Is there a formal e-mailbox for your team?

### DISCUSSION THREAD

**Discussion by fr1g** [SkriptLang/Skript#6034] [General]
I got some request that maybe sending email is better, and I'm not quite abled to use discord. So anyway to send your team an email? Or please point out what I've missed, if you already wrote your email address somewhere...

**Comment by Moderocky** [comment]
admin [at] skriptlang.org

**Comment by fr1g** [comment]
thanks a lot. i will try it outОтправлено с iPhone19 сент. 2023 г., в 15:14, Moderocky ***@***.***> написал(а):﻿
admin [at] skriptlang.org
